docs/getting-started.rst

```diff
@@ -105,56 +128,42 @@ properties of each retrieved Device object.
 
 Calling the SDK Directly
 ------------------------
+
 Now we'll repeat this example, but using the Python command line directly without a script.
 
-Access your Python interpreter with the ``python`` command (or ``python3`` if required) and type:
+Access your Python interpreter with the ``python`` command (or ``python3`` if required) and type::
 
->>> from cbc_sdk.rest_api import CBCloudAPI
->>> from cbc_sdk.platform import Device
->>> cb = CBCloudAPI(profile='default')
+    >>> from cbc_sdk.rest_api import CBCloudAPI
+    >>> from cbc_sdk.platform import Device
+    >>> cb = CBCloudAPI(profile='default')
 
 This imports the necessary classes and creates an instance of the base ``CBCloudAPI`` object.  By default, the file
 credentials provider is used. We set it to use the ``default`` profile in your ``credentials.cbc`` file, which you
 set up earlier.
 
-**N.B.:** On Windows, a security warning message will be generated about file access to CBC SDK credentials being
+.. note:: On Windows, a security warning message will be generated about file access to CBC SDK credentials being
 inherently insecure.
```

Comment:
This line break pushes the text outside the blue note box.

![Screenshot 2023-09-07 at 7 24 49 AM](https://github.com/carbonblack/carbon-black-cloud-sdk-python/assets/55465092/80aa232c-09a7-40fe-9e9b-c14bdf94110b)

Reply:
Fixed.